CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-# Contributing to [project-title]
+# Contributing to iOS (Swift) native authentication sample app
```

Comment:
We already updated this file, can you please take last version of "main" branch?